ObjCTraining/ContactDetailsViewController.h

```diff
@@ -0,0 +1,18 @@
+//
+//  ContactDetailsViewController.h
+//  ObjCTraining
+//
+//  Created by RWuser on 08/04/16.
+//  Copyright © 2016 RWuser. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface ContactDetailsViewController : UIViewController
+@property (weak, nonatomic) IBOutlet UILabel *name;
```

Comment:
https://github.com/WeezLabs/objective-c-style-guide#naming

It's better to nameLabel